app/views/handlers/individual_response.py

```diff
@@ -322,11 +323,10 @@ def _render_block(self) -> dict[str, Any]:
         )
 
     def _update_section_status(self, status: str) -> None:
-        self._questionnaire_store.progress_store.update_section_or_repeating_blocks_progress_completion_status(
+        self._questionnaire_store.progress_store.update_section_status(
             # Type ignore: Current usages of this method occur when Individual Section ID exists and is not None
```

Comment:
This is not needed anymore since type ignore got removed

Reply:
Done